docs/curriculum-topics-resources/open-source-licenses.md

```diff
@@ -4,75 +4,80 @@ sidebar_position: 2
-* The I.P. leads You (the company) and decision-making is aided --- maintain tighter controls on decision-making.
+* The I.P. leads you and decision-making is aided --- maintain tighter controls on decision-making.
 
-_**Examples of Copyleft Licenses**_: GPL-3.0 or other variants (LGPL, AGPL), Mozilla Public License (MPL), and a few others
+_**Examples of Copyleft Licenses**_: GPL-3.0 or other variants (LGPL, AGPL), Mozilla Public License (MPL), CC BY-SA, and a few others
 
-### Threats & Benefits to Copyleft Licensing Strategy
 **Threats:**
 * Not everyone likes copyleft. Big Tech may have incompatible business models, and governments/NGOs may have legacy tech that is difficult to comply with copyleft.
 * Requires up-front investment into community development.
+
 **Benefits:**
 * Rally external people together in a central community. The contributing dynamic changes because people are incentivized to contribute back.
 
 
 ## Software Licenses
-For software licenses, there are a number of acceptable licenses depending on your use case, including the GNU General Public license, the MIT license, or the BSD license. 
+For software licenses, there are a number of acceptable licenses depending on your use case, including the GNU General Public license, the MIT license, or the BSD license.
 
 ![](https://miro.medium.com/max/1400/0*Nbx3Uw8UgXQgHyzI.png)
 
-_**A licensee of BSD-licensed software can:**_
+## Hardware Licenses
+Open-source hardware refers to tangible objects such as machinery, mobile technology and other devices whose designs and other documentation for how to build the hardware are licensed under an open source license. Open source hardware designs, documentation, and bill of materials are available in a public repository so that they can be modified, changed or distributed.
+
+For hardware licenses, we recommend the **CERN, MIT, or TAPR Open Hardware Licenses**.
 
-* Use, copy and distribute the unmodified source or binary forms of the licensed program 
-* Use, copy and distribute modified source or binary forms of the licensed program provided that: 
-    * all distributed copies are accompanied by the license 
-    * the names of the previous contributors are not used to promote any modified versions without their written consent 
+For now, we will simply cover the CERN Open Hardware License, or OHL, developed by the CERN European Organization for Nuclear Research. It is to be included alongside the hardware design documentation files.
 
-We recommend this license because a licensee can take BSD-licensed code and add to it, safe in the knowledge that whatever they contribute **can be distributed in whatever way they choose.** For this reason the license is seen as friendly to traditional software business models that depend upon keeping the source private and capitalising on the sale of licensed binaries. 
```

Comment:
I removed the specific recommendation for BSD. Do we actually recommend BSD?

If you're looking at permissive licenses, I would recommend either Apache 2.0 or MIT over BSD. Apache 2.0 includes a patent clause, and MIT is much more readable. (again, not a lawyer)